src/Firehose.Web/Firehose.Web.csproj

```diff
@@ -59,6 +59,9 @@
     <Reference Include="System.Web">
       <Private>True</Private>
     </Reference>
+    <Reference Include="System.Web.Mvc, Version=5.2.3.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35">
```

Comment:
Is this reference needed? Seems unlikely that the site has changed and now requires this reference?